pages/auth/forgot-password.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AsetPoint | Forgot Password</title>
```

Comment:
I think you have misspelled in title --> aset

Reply:
Yes...thanks for pointing out.